src/navigation/search.py

```diff
@@ -54,7 +54,8 @@ class SearchStateTransitions(Enum):
 
     no_fiducial = "WaypointState"
     continue_search = "SearchState"
-    found_fiducial = "SingleFiducialState"
+    found_fiducial_post = "ApproachPostState"
```

Comment:
I think better names would be "found_post" and "found_one_post_gate" or something similar